### ISSUE TITLE
[BUG] Vertical hook pull is too weak

### ISSUE THREAD

**Issue by SlowDeepExhalation** [AoTTG-2/AoTTG-2#668]
**Describe the bug**
When your character hooks any object above or below them, they're pulled upwards or downwards towards that object.  The farther away the object, the harder your character is pulled.  If you land two hooks, the pull is increased substantially.

In AoTTG2, vertical hook pull is significantly weaker than it is in RC.  This is due both to the characters having incorrect ACL values, making them too heavy, and seemingly some further problem in the code--they're pulled less than even characters of their weight should be.

**To Reproduce**
1. Copy this map script and load it into RCMod/Anarchy/Your AoTTG 1 mod of choice in racing mode: https://pastebin.com/eaFnxWwJ
2. Spawn as Eren, who has 100 ACL.
3. Look up at a block above you, so that the reticle reports a distance of 101.  Tap (do not hold) a single hook key.
4. You should be pulled 79-80 units in the air, completing the "race" a single time at the very apex of your jump.  Any character with more ACL than Eren will travel higher, and be registered as completing the "race" twice.
5. Copy this map script and load it into AoTTG 2 in racing mode:  https://pastebin.com/A5MFkNiD
6. Spawn as any 100 ACL character (all of them, at the time of reporting), and repeat step 3.  You will not come even close to finishing.

**Expected behavior**
To put it simply, AoTTG2 should perform *identically* to AoTTG 1 mods.  It does not.  This is a complex issue however, and making them behave the same in only this specific example would be insufficient.

**Additional context**
Currently, it is impossible to empirically collect enough information about the details of this bug to fix it.  #624 must be completed first, before additional testing is possible to identify the exact issue.

A note to save this information where it is visible: According to Order (the developer of Anarchy mod), ACL determines character mass for hook pull by this formula: `mass = 0.5 - (Acl - 100) * 0.001;`
AoTTG2 should match this, though I expect the bug is more complex than a simple formula being wrong.

**Contributor instructions** (Do not modify)
1. Try to recreate the bug via the reported reproduction steps. If the bug can be reproduced, remove the investigation label. If the bug cannot be reproduced, please add a comment, attach "On Hold" label and inform the submitter that the bug cannot be reproduced. If there is no reply after 2 weeks, the issue can be closed.
2. Follow the default development team procedures (Create a new branch, apply your fix, create a pull request, submit new builds for testing team in dev squad)
3. Once testing team has confirmed the issue is resolved and the pull request is accepted, the issue can be closed.
